src/Extension/ExtendableTrait.php

```diff
@@ -566,7 +567,10 @@ protected function extensionGetParentClass(object $instance = null)
             return false;
         }
 
-        while (!in_array(ExtendableTrait::class, $parent->getTraitNames())) {
+        while (
+            !in_array(ExtendableTrait::class, $parent->getTraitNames())
+            && !in_array(OctoberExtendableTrait::class, $parent->getTraitNames())
```

Comment:
I could certainly profile both, but I reckon we'd see very minimal difference between the two. Reflection is very quick in PHP 8 now.

Reply:
If you'd be willing to profile both that would be appreciated, given how frequently this could be hit in a request I'd prefer to be better safe than sorry.

Reply:
The code as it is now (while loop calling `$parent->getTraitNames()` directly):

![Screenshot from 2024-03-08 09-36-59](https://github.com/wintercms/storm/assets/15900351/3a96c2a7-7a06-4981-b582-37587f819899)

The code updated to store the trait names first and check both traits against the variable:

![Screenshot from 2024-03-08 09-41-33](https://github.com/wintercms/storm/assets/15900351/7a4e2386-c2d3-404c-8453-bd4a722269e9)

Pretty much exactly the same performance exhibited by both.